server/gql/models/manufacturers.js

```diff
@@ -0,0 +1,77 @@
+import { GraphQLID, GraphQLNonNull, GraphQLObjectType, GraphQLString } from 'graphql';
+import { createConnection } from 'graphql-sequelize';
+import { productQueries } from './products';
+import { getNode } from '@gql/node';
+import db from '@database/models';
+import { sequelizedWhere } from '@database/dbUtils';
+import { totalConnectionFields } from '@utils/index';
+import { Op } from 'sequelize';
+
+const { nodeInterface } = getNode();
+
+export const manufacturerFields = {
+  id: { type: GraphQLNonNull(GraphQLID) },
+  name: { type: GraphQLString },
+  origin_country: { type: GraphQLString }
+};
+
+export const Manufacturer = new GraphQLObjectType({
+  name: 'Manufacturer',
+  interfaces: [nodeInterface],
+  fields: () => ({
+    ...manufacturerFields,
+    products: {
+      ...productQueries.list,
+      resolve: (source, args, context, info) =>
+        productQueries.list.resolve(source, args, { ...context, store: source.dataValues }, info)
+    }
+  })
+});
+
+export const ManufacturerConnection = createConnection({
+  nodeType: Manufacturer,
+  name: 'manufacturer',
+  target: db.manufacturers,
+  before: (findOptions, args, context) => {
+    findOptions.include = findOptions.include || [];
+    if (context?.product?.id) {
+      findOptions.include.push({
+        model: db.products,
+        where: {
+          id: context.product.id
+        }
+      });
+    }
+    if (args.likeName) {
```

Comment:
you will no longer need this, since you have added support for where in the input args